webapp/lib/app/nook/view.dart

```diff
@@ -69,9 +69,15 @@ class NookPageView extends PageView {
       showNormalStatus('signed in: ${latestCommitHash.substring(0, 8)}...');
     }, onError: (_) { /* Do nothing */ });
 
+    var backPageLink = new Element.a()
+      ..classes.add('configuration-view__back-link')
```

Comment:
the css for this class is defined only for the `configure` pages, not for converse, so the css for it won't actually apply in Nook.. I think this is a good candidate to generalise and move it into a top-level css file?